packages/validator/test/unit/services/attestation.test.ts

```diff
@@ -68,6 +71,7 @@ describe("AttestationService", function () {
           chainHeadTracker,
           syncingStatusTracker,
           null,
+          createChainForkConfig(config),
```

Comment:
we might wanna run these tests pre-/post-electra, can be addressed in the PR that enabled v1 apis pre-electra

Reply:
added post-electra tests in https://github.com/ChainSafe/lodestar/pull/7024